hazelcast/src/main/java/com/hazelcast/config/ServiceConfig.java

```diff
@@ -65,11 +65,11 @@ public ServiceConfig setClassName(final String className) {
         return this;
     }
 
-    public Object getServiceImpl() {
+    public Object getImplementation() {
```

Comment:
as much as I like this rename it's problematic because `ServiceConfig` is a (very) public class -> it breaks compatibility.